ReactiveObjCBridge/ObjectiveCBridging.swift

```diff
@@ -202,20 +219,20 @@ private final class RACSwiftScheduler: RACScheduler {
 	}
 }
 
-private func defaultNSError(_ message: String) -> NSError {
-	return Result<(), NSError>.error(message)
-}
-
-private func defaultNSError(_ message: String, file: String, line: Int) -> NSError {
-	return Result<(), NSError>.error(message, file: file, line: line)
-}
+//private func defaultNSError(_ message: String) -> NSError {
+//	return Result<(), NSError>.error(message)
+//}
+//
+//private func defaultNSError(_ message: String, file: String, line: Int) -> NSError {
+//	return Result<(), NSError>.error(message, file: file, line: line)
+//}
```

Comment:
This is this commented out?

Reply:
Yes, my mistake, forgot to remove it. I've updated the PR